src/Utils.ts

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import Constants from "./Constants";
+import { PathUtils } from "./PathUtils";
+import MetaProperty from "./MetaProperty";
+import { ModelManager } from "./ModelManager";
+
+export class Utils {
+    /**
+     * Returns all the tags for requested state
+     *
+     * @returns {String} all the tags for requested state
+     */
+    public static getTagsForState(state : string) : string {
+        let tags = '';
+        if (state === Constants.AUTHORING) {
+            const clientLibs = this.generateClientLibsUrl();
+            clientLibs.forEach(clientlib => {
+                if (clientlib.endsWith('.js')) {
+                    tags = tags.concat(this.generateElementString('script', 'src', clientlib));
```

Comment:
please convert this `script` and `stylesheet` strings into ENUMs that `generateElementString` can reuse